ZConfig/cmdline.py

```diff
@@ -155,7 +157,7 @@ def get_section_info(self, type, name):
             bk = self.basic_key(s, pos)
             if name and self._normalize_case(s) == name:
                 L.append((optpath[1:], val, pos))
-            elif bk == type:
+            elif bk == type: # pragma: no cover
```

Comment:
FWIW when I was reviewing this, I wanted to ask why this comparison wasn't using `if bk is type` to compare with the `type` builtin.  Then I thought a bit more and decided maybe `type` was a local variable shadowing the builtin.

(Also, given that my Vim syntax-highlights `type` in a different color from regular variables makes me want to use `type_` instead of shadowing.)